### PR TITLE
Style: uninstall.sh (Tabs to spaces)

### PR DESCRIPTION
**Pull request by rrobgill**
**By submitting this pull request, I confirm the following:** 

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [ ] I have tested my proposed changes, and have included unit tests where possible.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**
Standardize indentation using spaces instead tabs.

uninstall.sh contains a mixture of tabs and spaces used for indentation in different parts of the file.

**How does this PR accomplish the above?:**
Everywhere that used tabs has been converted to spaces, compatible with the indentation style used in automated\ install/basic_install.sh

**What documentation changes (if any) are needed to support this PR?:**
None. 

---

Aside from adjustments to indent spacing, stickler-bot raised some issues with the old code, which have been fixed up too.

### CODE REVIEW

**Comment by rrobgill** [comment]
Oh Stickler-bot, that's not even my code...

**Comment by Mcat12** [comment]
Can you use 4 spaces for each indentation?

**Comment by rrobgill** [comment]
@Mcat12 Everything has been converted to 4 spaces per indent, no tabs. I made the same changes to the code added in #2167 (which used tabs).

**Comment by Mcat12** [comment]
Instead of making the changes from another PR, merge `development` into your branch.

File: automated install/uninstall.sh [hunk 1]
```diff
@@ -48,167 +48,157 @@ distro_check
...
-	${SUDO} rm -f /usr/local/bin/pihole &> /dev/null
-	${SUDO} rm -f /etc/bash_completion.d/pihole &> /dev/null
-	${SUDO} rm -f /etc/sudoers.d/pihole &> /dev/null
-  echo -e "  ${TICK} Removed config files"
-
-  # Restore resolved 
-  if [[ -e /etc/systemd/resolved.conf.orig ]]; then
-    ${SUDO} cp /etc/systemd/resolved.conf.orig /etc/systemd/resolved.conf
-    systemctl reload-or-restart systemd-resolved
-  fi
```
[the remark this answers is not in the document]
Comment (rrobgill) [reply automated install/uninstall.sh]: missed that in the merge.
Replaced now.

[the remark this answers is not in the document]
Comment (Mcat12) [reply automated install/uninstall.sh]: It's not in the same place

[the remark this answers is not in the document]
Comment (rrobgill) [reply automated install/uninstall.sh]: Sorry, misread the line numbers as where it needed to be, not where it came from.
Really fixed this time.

**Pull request merged**